widget/embedded/src/components/ConfirmWalletsModal/ConfirmWalletsModal.tsx

```diff
@@ -438,9 +459,22 @@ export function ConfirmWalletsModal(props: PropTypes) {
                         }>
                         <Divider size={4} />
                         <StyledTextField
+                          show={showCustomDestination}
+                          ref={customDestinationInputRef}
+                          style={{ padding: 0 }}
                           autoFocus
                           placeholder={i18n.t('Your destination address')}
                           value={customDestination || ''}
+                          suffix={
+                            !isFirefox &&
+                            (!isMobile || !customDestination) && (
```

Comment:
Is this condition accurate?
`(!isMobile || !customDestination)`

Reply:
Yes, in mobile view we should show the button only if the user has not entered any value.